src-backend/api/signals/page_pre_save.py

```diff
@@ -0,0 +1,18 @@
+from django.dispatch import receiver
+from django.db.models.signals import pre_save
+from django.db.models import F
+from api.models import Page
+import logging
+
+
+logger = logging.getLogger('debugger')
```

Comment:
According to https://docs.djangoproject.com/en/1.8/topics/logging/#django-s-default-logging-configuration it seems like these logs will only be logged on debug mode, but @Trinovantes can confirm for sure.